src/Compilers/Core/Portable/FileSystem/PathUtilities.cs

```diff
@@ -735,6 +735,16 @@ public static string NormalizePathPrefix(string filePath, ImmutableArray<KeyValu
             return filePath;
         }
 
+        public static string NormalizeDriveLetter(string filePath)
```

Comment:
Is there a reason why we have separate normalize helpers vs. putting them all in one method? Basically I'm wondering if there is ever a case where it's okay to call only some of these normalized methods but not othhers.

Reply:
To my understanding the section name storage from parsing only needs to call this one, while retrieval of options for a filepath should call both. @jjonescz is that correct?

Reply:
It's correct that calling just the one for section names fixes the bug at hand. But I agree with Jared that we could call both even for section names, I imagine that would fix some more inconsistencies. For example, a section `[*//*.cs]` doesn't match file `a/b.cs` today probably. I'm not sure it should per editorconfig spec if there is such a thing though!

Reply:
I'm happy for issue + follow up.

Reply:
https://github.com/dotnet/roslyn/issues/73620